apps/src/signUpFlow/loginTypeSelection.module.scss

```diff
@@ -113,6 +117,7 @@ body {
     .googleButton {
       background-color: $google_brand_color;
       margin-bottom: .5rem;
+      color: $white;
```

Comment:
You might be able to refactor this to group shared button styles using the `button` selector (or add a second repeating class to each button) for things like margin, text color, font-weight etc. Or even put them in a container and use flexbox for the spacing between them.

Reply:
Yes! Is it alright if I wait for Turner's big refactor to do more specific style cleanup?

Reply:
Oh yes absolutely! I can also do a final css pass before we launch 🙌